code/modules/chemistry/tools/dispensers.dm

```diff
@@ -562,6 +562,38 @@
 			return ..()
 
 
+/obj/reagent_dispensers/still/shittystill
+	name = "shitty still"
+	desc = "Looks like it some sort of makeshift distillery, made out of salvaged parts."
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "shittystillFIN"
+	amount_per_transfer_from_this = 20
+	var/list/random_reagents_list = list("barium","flourine","lithium","magnesium","mercury",\
+"oxygen","phosphorus","plasma","potassium","radium","silver","sugar","hydrogen","water",\
+"carbon","chlorine","sulfur","calcium","iron","ethanol")
```

Comment:
Maybe add some fun reagents as well, ala discount dan; with a lesser chance and smaller amount?

Reply:
Definitely a planned feature. Will probably add something in a separate PR alongside some additional stuff

Reply:
Makes sense!